packages/react-sketch-canvas/src/Canvas/index.tsx

```diff
@@ -59,6 +60,7 @@ export const Canvas = React.forwardRef<CanvasRef, CanvasProps>((props, ref) => {
   const {
     paths,
     isDrawing,
+    scrollOnTouch,
```

Comment:
Users must not be able to set `allowOnlyPointerType` as touch or all while they set `scrollOnTouch`. ReactSketchCanvas must throw an error with some meaningful error message to avert users from running into this scenario. CanvasProps must also be updated to be conditional here.